2-mandatory/3-financial-times.js

```diff
@@ -4,34 +4,72 @@
     The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
+
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+     let newArray = [];
+    for (let i = 0; i < allArticleTitles.length; i++){
+        if (allArticleTitles[i].length <= 65) {
+            newArray.push(allArticleTitles[i])
+        }
+    }
+    return newArray;
 }
+ 
 
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
+
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+     let array = allArticleTitles[0].split(" ").length;
+    let spaceCount;
+    for (let i = 0; i < allArticleTitles.length; i++) {
+        wordCount = allArticleTitles[i].split(" ").length;
```

Comment:
I suggest to add .trim() method  before split ,Because if there is any space at first or in the end of string it will wrongly add to the number of words